scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -704,6 +767,54 @@ def load_multihead_quantized_softmax_receipt_payload(
     return copy.deepcopy(_load_multihead_quantized_softmax_receipt_payload(path, run_native))
 
 
+@functools.lru_cache(maxsize=2)
+def _load_longseq_fused_softmax_payload(path: pathlib.Path, run_native: bool) -> dict[str, Any]:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)